src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/AndroidUpdateResourcesTest.cs

```diff
@@ -830,6 +830,7 @@ public class Foo {
 					rawToDelete,
 				},
 			};
+			libProj.SetProperty ("Deterministic", "true");
```

Comment:
@jonathanpeppers I'm not sure why I've had to add this to this particular test. It was failing without it.

Reply:
Frankly, we should use `$(Deterministic)`=true *everywhere*…